techAssignment/src/App.vue

```diff
@@ -1,78 +1,107 @@
 <script>
+import InputCommand from "./components/InputCommand.vue";
+import CommandHistory from "./components/CommandHistory.vue";
+
 export default {
+  name: "App",
+
   data() {
     return {
-      count: 0,
+      commandInput: "",
+      commandInputArray: [],
```

Comment:
`this.commandInputArray` is not used in this component, I think we could remove it.